src/classes/GE_PaymentServices.cls

```diff
@@ -83,6 +94,16 @@ public with sharing class GE_PaymentServices {
         config = new Configuration().keyValueMap;
     }
 
+    @AuraEnabled(cacheable=true)
+    public static String getPaymentTransactionStatus() {
+        Map<String, String> paymentStatusNameByEnum = new Map<String, String>();
+
+        for (ElevateTransactionStatus status : ElevateTransactionStatus.values()) {
+            paymentStatusNameByEnum.put(status.name(), status.name());
+        }
+        return JSON.serialize(paymentStatusNameByEnum);
+    }
```

Comment:
Is there a use-case for building and returning a key-value pair map here when the keys and values are always the same value?  Could this method just return the list of values?

`return ElevateTransactionStatus.values();`

Reply:
@kenlewis-sfdo This is mainly to preserve the object structure that an `enum` would have if created in Javascript so that we can reference a property as a key and get the string value corresponding to that property.

Reply:
Ok.  Does the code need that structure on the LWC side?  Or does it just validate that a status received from the service is one of the expected valid values, in which case this list could just be a set instead of a map.  

Also, I don't see this new `PAYMENT_TRANSACTION_STATUS_ENUM` being used anywhere currently - is this a placeholder?

Reply:
@kenlewis-sfdo yeah I removed that unused constant. We actually need to explicitly reference enum values in the object instead of performing a `contains` operation so that's why we need to return an object vs an array.